# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux-firmware/no-symlink-check.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-firmware-20241110/copy-firmware.sh.vanilla	2024-11-14 16:49:18.017061474 +0100
+++ linux-firmware-20241110/copy-firmware.sh	2024-11-14 16:49:37.693728498 +0100
@@ -99,7 +99,7 @@
 done
 
 # Verify no broken symlinks
-if test "$(find "$destdir" -xtype l | wc -l)" -ne 0 ; then
+if false && test "$(find "$destdir" -xtype l | wc -l)" -ne 0 ; then
     err "Broken symlinks found:\\n$(find "$destdir" -xtype l)"
 fi
 
